.github/workflows/update-docs.yml

```
@@ -0,0 +1,48 @@
+name: Update Template Documentation
+
+on:
+  schedule:
+    - cron: '0 16 * * 2' # Every Tuesday at 11 AM EST (16:00 UTC)
+  workflow_dispatch:
+
+permissions:
+  contents: write
```

```suggestion
  contents: write
  pull-requests: write
```

This is needed for creating the PR, `contents: write` just allows branch creation

---

I see. I thought they were same. Fixed it. Thanks.